features/em.yml

```
@@ -1,7 +1,6 @@
-draft_date: 2024-06-05
 name: <em>
-description: The `<em>` element TODO.
+description: The `<em>` HTML element emphasizes or stresses the content, rendering text as italic by default.
```

```suggestion
description: The `<em>` HTML element emphasizes or stresses the content, styling text as italic by default.
```

@foolip asked me to chime in here, on the convention we're going to establish via https://github.com/web-platform-dx/web-features/pull/1403. I'm putting forward "styling" instead of "rendering" because I'd like to emphasize that this is a style that can be overridden by the page author and not solely the domain of the browser's rendering engine.